infrastructure/operators.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: default
+  vars:
+    users:
+    - username: tomlee
+      shell: /bin/bash
+      github: user512
+    - username: zee
+      shell: /bin/bash
+      github: zspencer
+
+  become: yes
+  become_method: sudo
+  tasks:
+  - name: "make sure SSH is allowed"
+    ufw:
+      rule: allow
+      port: 22
+      proto: tcp
```

Comment:
I think the UFW is already configured to allow port 22; we probably don't need this?